FYDP_ModV1/src/version1.java

```diff
@@ -17,12 +184,14 @@ public static void modelConfig() {
 		int awaySubject = n-2;
 		
 	//define parameters - teachers
-		int n2 = 16;
-		double [] FTE = {1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.2,0.2,0.6,0.2,1.0,1.0};
-		int frenchTeachlb = n2-2;
+		//Hardcoded values 
+		
+//		int n2 = 16;
+//		double [] FTE = {1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.2,0.2,0.6,0.2,1.0,1.0};
+//		int frenchTeachlb = n2-2;
 		int frenchTeachub = n2-1;
 		int frenchTeach = 2;
-		String [] teacherNames;
+//		String [] teacherNames;
 	
 	//define parameters - cohorts
 		int n3 = 13;
```

Comment:
This needs to be variable, its messing up printing 'frenchnum', also confused about the difference between frenchnum and frenchteachlb as all we need to do is collect the number of french teachers, but this could just be a point of clarification on my end

Reply:
french num is dependent on N3 which is taken in on a different page and should be dynamically updated once we have page two and french teachers are listed at the end of the teacher list